windows/src/support/charident/Makefile

```diff
@@ -13,7 +13,11 @@ clean: def-clean
 signcode:
     $(SIGNCODE) /d "Character Identifier" $(PROGRAM)\support\charident.exe
 
+wrap-symbols:
+    $(SYMSTORE) $(PROGRAM)\support\charident.exe /t keyman-windows
+    #TODO: $(SYMSTORE) $(DEBUGPATH)\support\charident.dbg /t keyman-windows
+
 backup:
-    $(WZZIP) $(BUILD)\online\charident.zip  $(BACKUPDEFAULTS) charident.exe
+    $(WZZIP) $(BUILD)\support\charident.zip  $(BACKUPDEFAULTS) charident.exe
```

Comment:
I'm guessing this is a thrown-in typo fix?

Reply:
Ah yes, I noted one of the other typo fixes but missed this one. The whole `backup` pattern will hopefully go away at some point (we still kinda need it for .tds symbols at present).